lighthouse-core/audits/server-response-time.js

```diff
@@ -11,27 +11,27 @@ const MainResource = require('../computed/main-resource.js');
 
 const UIStrings = {
   /** Title of a diagnostic audit that provides detail on how long it took from starting a request to when the server started responding. This descriptive title is shown to users when the amount is acceptable and no user action is required. */
-  title: 'Server response times are low (TTFB)',
+  title: 'Server response times are low',
   /** Title of a diagnostic audit that provides detail on how long it took from starting a request to when the server started responding. This imperative title is shown to users when there is a significant amount of execution time that could be reduced. */
-  failureTitle: 'Reduce server response times (TTFB)',
+  failureTitle: 'Reduce server response times',
```

Comment:
```suggestion
  failureTitle: 'Reduce server response time',
```

feels weird to call this plural when we're only ever flagging a single duration, I guess there's an argument to be made for the RUM aggregate response times, but that's not really Lighthouse's bag

Reply:
> feels weird to call this plural when we're only ever flagging a single duration, I guess there's an argument to be made for the RUM aggregate response times, but that's not really Lighthouse's bag

Yeah, agreed that is weird. I'll change it.

What do you think of `Reduce initial server response time`? Is "initial" overkill or clearer what's being measured?

Reply:
ya that works for me 👍 

I was kinda relying on the description to make that distinction clear since everything I came up with around `Reduce document server repsonse time` sounded way too awkward to my ears haha, but this is a nice compromise :)